src/contexts/AuthProvider.js

```diff
@@ -79,40 +80,46 @@ export const Auth0Provider = ({ children }) => {
         config
       );
 
-      const data = await response.json();
+      let data = null;
+      try {
+        data = await response.json();
+      } catch (err) {}
```

Comment:
Not sure if this is best practice... is there a reason you didn't make the scope of the try/catch on the entire function?